cmd/buildah/common.go

```diff
@@ -148,14 +149,14 @@ func systemContextFromOptions(c *cli.Context) (*types.SystemContext, error) {
 
 func parseCreds(creds string) (string, string, error) {
 	if creds == "" {
-		return "", "", errors.Wrapf(syscall.EINVAL, "credentials can't be empty")
+		return "", "", nil
```

Comment:
just noted we're always returning a nil error, I"m going to yank that out.  New patch shortly.